src/js/api/bulk-order.js

```diff
@@ -0,0 +1,507 @@
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request, null, result));
+        });
+    });
+};
+
+/**
+ * Sets the bulk order status to 'ready'.
+ *
+ * @param {Number} number The number of the bulk order to update.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.readyBulkOrder = function(number, options, callback) {
+    return this.setBulkOrderStatus(number, "ready", options, callback);
+};
+
+/**
+ * Sets the bulk order status to 'ready'.
+ *
+ * @param {Number} number The number of the bulk order to update.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {Promise} The update bulk order.
+ */
+ripe.Ripe.prototype.readyBulkOrderP = function(number, options) {
+    return new Promise((resolve, reject) => {
+        this.readyBulkOrder(number, options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request, null, result));
+        });
+    });
+};
+
+/**
+ * Sets the bulk order status to 'ready'.
+ *
+ * @param {Number} number The number of the bulk order to update.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.sendBulkOrder = function(
+    number,
+    trackingNumber,
+    trackingUrl,
+    options,
+    callback
+) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    options = Object.assign(options, {
+        params: {
+            tracking_number: trackingNumber,
+            tracking_url: trackingUrl
+        }
+    });
+    return this.setBulkOrderStatus(number, "send", options, callback);
+};
+
+/**
+ * Sets the bulk order status to 'ready'.
+ *
+ * @param {Number} number The number of the bulk order to update.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {Promise} The update bulk order.
+ */
+ripe.Ripe.prototype.sendBulkOrderP = function(number, trackingNumber, trackingUrl, options) {
+    return new Promise((resolve, reject) => {
+        this.sendBulkOrder(
+            number,
+            trackingNumber,
+            trackingUrl,
+            options,
+            (result, isValid, request) => {
+                isValid ? resolve(result) : reject(new ripe.RemoteError(request, null, result));
+            }
+        );
+    });
+};
+
+/**
+ * Sets the bulk order status to 'blocked'.

```

Comment:
Maybe `to 'block'` instead? I know it's how it's described [here](https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/order.js#L837) but maybe would make more sense to use the status name.

Reply:
I think it is `blocked` since it will be the status of the order after this request https://github.com/ripe-tech/ripe-core/blob/761b108379ceec1e2fdd820465ae5ab2defae708/src/ripe_core/models/order.py#L78